middlewares/replace_path.go

```diff
@@ -18,3 +18,8 @@ func (s *ReplacePath) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	r.URL.Path = s.Path
 	s.Handler.ServeHTTP(w, r)
 }
+
+// SetHandler sets handler
+func (s *ReplacePath) SetHandler(Handler http.Handler) {
```

Comment:
It should not be in use since you've added it only but there's no call happening.

If I'm wrong, the CI will tell us. :-)